### PR TITLE
Update changelog and authors for unyt 2.9.0

### PR DESCRIPTION
**Pull request by jzuhone**
This PR updates the changelog and authors for an upcoming release of unyt 2.9.0.

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,63 @@
 History
 =======
 
+2.9.0 (2022-07-13)
+------------------
+
+* Dropped support for Python 3.6 and 3.7.
```
Comment (neutrinoceros) [inline HISTORY.rst:8]:
```suggestion
* Dropped support for Python 3.5, 3.6 and 3.7.
```

File: HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,63 @@
...
+* Added support for Python 3.9 and 3.10.
```
Comment (neutrinoceros) [inline HISTORY.rst:9]:
```suggestion
* Added support for Python 3.8, 3.9 and 3.10.
```

File: HISTORY.rst [hunk 3]
```diff
@@ -2,6 +2,63 @@
...
+  <https://github.com/yt-project/unyt/pull/180>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* Add a method to parse a ``unyt_quantity`` from a string expression. See `PR #191
+  <https://github.com/yt-project/unyt/pull/191>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* Fix an issue where a ``unyt_array`` with dtype int8 could not be converted
+  to a different unit. See `PR #197
+  <https://github.com/yt-project/unyt/pull/197>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* The import time for ``unyt`` has been optimized. See `PR #199
```
Comment (neutrinoceros) [inline HISTORY.rst:29]: Modesty++
Also this particular PR should probably be mentioned for the actual change that's performed. It's still (remotely) possible that it's breaking to someone.
```suggestion
* The import time for ``unyt`` has been reduced. See `PR #199
```

**Review by neutrinoceros — COMMENTED** [review]
Thank you John for preparing this, here are a couple remarks

File: HISTORY.rst (hunk 1 above)
Comment (neutrinoceros) [reply HISTORY.rst:8]: I see that 3.5 support was "dropped" already in version 2.8, according to the changelog, however I confirm that Python 3.5 was still documented as supported according to Python classifiers and `python_requires`. See 
https://github.com/yt-project/unyt/blob/eeefa00a2fddbf0dba6ab854e968ef43e31f851e/setup.py#L51

Comment (jzuhone) [reply HISTORY.rst:8]: To reduce confusion, I'm going to leave this line as-is.

Comment (neutrinoceros) [reply HISTORY.rst:8]: Fine by me

**Comment by jzuhone** [comment]
Ok, I think this is ready to go.

File: HISTORY.rst [hunk 4]
```diff
@@ -2,6 +2,63 @@
...
+* Add the Rydberg constant ``R_∞`` and unit ``Ry``, add the dimension
+  ``angular_frequency`` and the unit ``rpm``, and increase the precision of
+  Avogadro's number. See `PR #228 <https://github.com/yt-project/unyt/pull/228>`_.
+* Fix an issue where ``np.divide.reduce`` returns incorrect units for ``unyt_array``
+  instances. See `PR #230 <https://github.com/yt-project/unyt/pull/230>`_.
+  Thank you to Kyle Oman (@kyleaoman on GitHub) for the contribution.
+* Fix a regression where ``unyt_array.reshape`` wasn't wrapping
+  ``numpy.ndarray.reshape`` correctly. See `PR #242
+  <https://github.com/yt-project/unyt/pull/242>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
```
Comment (neutrinoceros) [inline HISTORY.rst:59]: Not sure this is worth mentioning in the changelog: this regression was never released, so it's not a diff wrt to 2.8.0

**Comment by neutrinoceros** [comment]
Please give me a few minutes to read through all of it

File: HISTORY.rst [hunk 5]
```diff
@@ -2,6 +2,63 @@
...
+* Fix typos in constants and unit prefixes. See `PR #218
+  <https://github.com/yt-project/unyt/pull/218>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* Fix an issue where multiplying a 1-element ``unyt_array`` would return a
+  ``unyt_quantity``. See `PR #225 <https://github.com/yt-project/unyt/pull/225>`_.
+  Thank you to Clément Robert (@neutrinoceros on GitHub) for the contribution.
+* Add the Rydberg constant ``R_∞`` and unit ``Ry``, add the dimension
+  ``angular_frequency`` and the unit ``rpm``, and increase the precision of
+  Avogadro's number. See `PR #228 <https://github.com/yt-project/unyt/pull/228>`_.
+* Fix an issue where ``np.divide.reduce`` returns incorrect units for ``unyt_array``
```
Comment (neutrinoceros) [inline HISTORY.rst:53]: for tense consistency
```suggestion
* Fix an issue where ``np.divide.reduce`` would return incorrect units for ``unyt_array``
```

**Review by neutrinoceros — COMMENTED** [review]
looks good, only two very minor remarks

File: HISTORY.rst (hunk 4 above)
Comment (jzuhone) [reply HISTORY.rst:59]: If it's ok with you, we can just remove it then

**Review by neutrinoceros — APPROVED** [review]

**Pull request merged**